apprentice_teacher.rb

```diff
@@ -1,21 +1,9 @@
-class ApprenticeTeacher
-  attr_reader :age, :salary, :phase, :target_raise
-  attr_accessor :name
+require_relative 'refactored_all_teachers'
 
+class ApprenticeTeacher < AllTeachers
   def initialize(options={})
-    @age = options.fetch(:age, 0)
-    @name = options.fetch(:name, "")
+    super
     @target_raise = 800
```

Comment:
Looks like this attribute is for an apprentice and teacher, how can this be DRY'd up?